src/CommunityToolkit.Aspire.Hosting.Apache.Tika/CommunityToolkit.Aspire.Hosting.Apache.Tika.csproj

```
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <AdditionalPackageTags>hosting adminer</AdditionalPackageTags>
```

```suggestion
    <AdditionalPackageTags>hosting tika</AdditionalPackageTags>
```

I assume that's a better set of tags